File: sample/src/main/java/com/github/kazy1991/prefeditor/sample/SampleOneSchema.java
package com.github.kazy1991.prefeditor.sample;


import com.github.kazy1991.prefkit.annotation.PrefKey;
import com.github.kazy1991.prefkit.annotation.PrefSchema;


@PrefSchema("SampleOneSchema")
public interface SampleOneSchema {

    String featureFlag = "featureFlag";

    @PrefKey(featureFlag)
    void setFeatureFlag1(boolean flag);


}

File: sample/src/main/java/com/github/kazy1991/prefeditor/sample/SampleOneSchema.kt
package com.github.kazy1991.prefeditor.sample


import com.github.kazy1991.prefkit.annotation.PrefKey
import com.github.kazy1991.prefkit.annotation.PrefSchema


@PrefSchema("SampleOneSchema")
interface SampleOneSchema {

    @PrefKey(featureFlag)
    fun setFeatureFlag1(flag: Boolean)

    companion object {
        const val featureFlag = "featureFlag"
    }
}
